developer/cloud/keyboard_info/1.0/index.php

```diff
@@ -332,6 +332,17 @@
   <p><b>object</b>, with each property an id of a keyboard and corresponding value is a <b>KeyboardRelatedInfo</b>.</p>
 </td>
 </tr>
+<tr>
+<td>deprecated</td>
+<td></td>
+<td></td>
+<td></td>
```

Comment:
```suggestion
<td>yes</td>
```

Since the deprecated field is "always generated automatically"